tests/common/type_coverage.h

```diff
@@ -158,4 +158,62 @@ void for_all_types_and_vectors(const named_type_pack<types...>& typeList,
   static_cast<void>(packExpansion);
 }
 
+
+/**
+ * @brief Run action for type, vectors amd marrays of this type
+ * @tparam action Functor template for action to run
+ * @tparam T Type to instantiate functor template with
+ * @tparam actionArgsT Parameter pack to use for functor template instantiation
+ * @tparam argsT Deduced parameter pack for arguments to forward into the call
+ * @param args Arguments to forward into the call
+ */
+template <template <typename, typename...> class action, typename T,
+          typename... actionArgsT, typename... argsT>
+void for_type_vectors_marray(argsT&&... args) {
+  if constexpr (std::is_same<T, bool>::value)
```

Comment:
```suggestion
  if constexpr (std::is_same<T, bool>::value) {
```

I'd prefer `if` and other statements wrapped in braces to prevent mistakes in further PRs

Reply:
Thanks, added.
+ added missing **type_name_string** specialization for **sycl::marray** in last commit https://github.com/KhronosGroup/SYCL-CTS/pull/140/commits/f27d955c6d256c6f032eaad856d5c95bb26085a8